Golem/Golem.cs

```diff
@@ -238,10 +238,34 @@ public async Task Stop()
             SafeCancel(_providerCancellationtokenSource);
             SafeCancel(_yagnaCancellationtokenSource);
 
-            await Provider.Stop(providerTimeout);
-            await Yagna.Stop(yagnaTimeout);
-
-            Status = GolemStatus.Off;
+            try
```

Comment:
I got once `Error` status when doing `Golem.Stop` on an instance with dead Provider, so I made it to catch `SystemError`, and continue stopping which will results with status  set to `Off`.
In case of a different Exception stopping will end, and status will be set to `Error`.